containers/src/Data/IntMap/Internal.hs

```diff
@@ -2301,37 +2301,37 @@ deleteFindMax = fromMaybe (error "deleteFindMax: empty map has no maximal elemen
 deleteFindMin :: IntMap a -> ((Key, a), IntMap a)
 deleteFindMin = fromMaybe (error "deleteFindMin: empty map has no minimal element") . minViewWithKey
 
+lookupMinSure :: IntMap a -> (Key, a)
+lookupMinSure (Tip k v)   = (k,v)
+lookupMinSure (Bin _ l _) = lookupMinSure l
+lookupMinSure Nil         = error "lookupMinSure Nil"
+
```

Comment:
Does worker-wrapper eliminate the `I#` wrappers here? I imagine so, with sufficiently recent GHC, but it might be worth checking and noting.

Reply:
Yes! The worker returns a `Int#`. Mentioned previously in https://github.com/haskell/containers/pull/976#discussion_r1451805774, but I checked it again and it's as expected.

Reply:
Excellent. Could you add a comment to that effect? I believe that's likely only true in recentish GHC, with nested CPR. Along with the two accidental space characters, I think that's all that's left to do, as far as I'm concerned.

Reply:
Hmm you're right, I wasn't aware that this is a somewhat recent change. I checked on 9.2 and it's returning a boxed Int.
I feel like we should use a `KeyValue !Int a` for the benefit of older GHCs, after all we're already doing that for `Map`. What do you think?

Reply:
Updated to use `KeyValue`.